src/Pos/Wallet/WalletMode.hs

```diff
@@ -153,13 +152,13 @@ instance MonadIO m => MonadTxHistory (WalletDB m) where
     getTxHistory = Tagged $ \addr _ -> do
         chain <- WS.getBestChain
         utxo <- WS.getOldestUtxo
-        res <- fmap (fst . fromMaybe (error "deriveAddrHistory: Nothing")) $
+        _ <- fmap (fst . fromMaybe (error "deriveAddrHistory: Nothing")) $
             runMaybeT $ flip runUtxoStateT utxo $
             deriveAddrHistory addr chain
-        pure undefined
+        pure $ error "undefined"
```

Comment:
`error` has `HasCallStack` constraint. So it will give module name and location. I think.